docs/implementations.rst

```diff
@@ -121,16 +137,42 @@ All network implementations are located in `mockturtle/networks/`:
 +--------------------------------+-------------+-------------+-------------+-----------------+
 | ``index_to_node``              | ✓           | ✓           | ✓           | ✓               |
 +--------------------------------+-------------+-------------+-------------+-----------------+
+| ``index_to_ci``                |             |             |             |                 |
```

Comment:
I am worried that these names are confusing, since index has some specific meaning in the network interface, which is an integer between `0` and `size() - 1` to address a note.

I suggest the following alternative names:
* `ci_at`
* `ci_from_index`

The latter, although slightly more verbose, can also be used for other methods, e.g., `ci_from_name` and in the current list of methods `ri_from_ro`. Then we can use `ri_from_ro(ro_from_index(i))` instead of having a special method for that.